docs/tutorials/cics-deploy/CICSDeployPlugin.md

```diff
@@ -5,7 +5,7 @@ Before you begin, [set up](../Setup.md) your environment to install a plug-in.
 ## Overview
 This tutorial covers installing and running this bundled Zowe CLI cics-deploy plugin as-is (without modification). 
 
-The plug-in adds a command to the CLI that supports generation and deployment of CICS Bundle resources from the current directory of your PC.
+The plug-in adds a command to the CLI that supports generation and deployment of CICS bundle resources from the current directory of your PC.
```

Comment:
Suggest PC -> workstation

Reply:
I agree that workstation is better. I'm not sure where PC came from, it might have been inherited from the sample plugin, or maybe from the text in one of the boxnotes.